### DISCUSSION TITLE
Rewrite the directions.

### DISCUSSION THREAD

**Discussion by scottonanski** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#884] [General]
They're out of date.

**Comment by sumeshir26** [comment]
How?

**Reply by LatridellActiveX** [reply]
The pages on the cloud console and credentials as well as Oauth credentials now have a slightly different layout compared to when the instructions were made. I can link detailed information if you would like. I was able to set it up regardless.

**Comment by KeyboardZombie** [comment]
seconded. google doesnt let you download ouath in the way the tutorial says anymore. its confusing.

**Reply by stOneskull** [reply]
how _do_ you download the oauth file?

**Comment by BogdanBTO** [comment]
верно, инструкция устарела, но я смог справиться

**Reply by lolouthefox** [reply]
> верно, инструкция устарела, но я смог справиться

In english?
